runtime/src/main/kotlin/kotlin/native/ThrowableExtensions.kt

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright 2010-2019 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the LICENSE file.
+ */
+package kotlin.native
+
+public fun Throwable.getStackTraceAddresses(): List<Long> =
```

Comment:
Please don't forget to add a kdoc summary for public API.

Reply:
Thanks for noticing.
See #3364.